prefect_docker/scripts/configuration/settings.toml

```diff
@@ -1,30 +1,38 @@
 [default]
+ELSST_SKOSMOS_URL="https://thesauri.cessda.eu"
 
 [development]
 debug = true
-
-DATAVERSE_URL="http://dataverse:8080"
-
+DOI_MINTER_URL = "http://datacite-minter:9999"
 DATAVERSE_IMPORTER_URL="http://dataverse-importer:8091"
 PUBLICATION_DATA_UPDATER_URL="http://publication-date-updater:8093"
 METADATA_FETCHER_URL="http://metadata-fetcher:8092"
 DATAVERSE_MAPPER_URL="http://dataverse-mapper-v2:8094"
 DANS_TRANSFORMER_SERVICE="https://transformer.labs.dans.knaw.nl"
 EMAIL_SANITATION_SERVICE="https://emailsanitizer.labs.dans.knaw.nl/sanitize"
 METADATA_REFINER_URL="http://metadata-refiner:7878"
+SEMANTIC_API_URL= "http://semanticapi:80/importdoi"
 
-BUCKET_NAME = "harvested-metadata"
+[staging]
+debug = false
+DOI_MINTER_URL = "https://dataciteminter.labs.dans.knaw.nl/submit-to-datacite/draft"
+DATAVERSE_IMPORTER_URL="https://dataverse-importer.labs.dans.knaw.nl"
+PUBLICATION_DATA_UPDATER_URL="https://dataverse-date-updater.labs.dans.knaw.nl"
+METADATA_FETCHER_URL="https://dataverse-fetcher.labs.dans.knaw.nl"
+DATAVERSE_MAPPER_URL="https://dataverse-mapper.labs.dans.knaw.nl"
+DANS_TRANSFORMER_SERVICE="https://transformer.labs.dans.knaw.nl"
+EMAIL_SANITIZER_URL="https://emailsanitizer.labs.dans.knaw.nl/sanitize"
+METADATA_REFINER_URL="https://metadata-refiner.labs.dans.knaw.nl"
+SEMANTIC_API_URL="http://semanticapi:80/importdoi"
 
 [production]
 debug = false
-
-DATAVERSE_URL="https://portal.staging.odissei.nl"
+DOI_MINTER_URL = "https://dataciteminter.labs.dans.knaw.nl/submit-to-datacite/register"
 DATAVERSE_IMPORTER_URL="https://dataverse-importer.labs.dans.knaw.nl"
 PUBLICATION_DATA_UPDATER_URL="https://dataverse-date-updater.labs.dans.knaw.nl"
 METADATA_FETCHER_URL="https://dataverse-fetcher.labs.dans.knaw.nl"
 DATAVERSE_MAPPER_URL="https://dataverse-mapper.labs.dans.knaw.nl"
 DANS_TRANSFORMER_SERVICE="https://transformer.labs.dans.knaw.nl"
 EMAIL_SANITIZER_URL="https://emailsanitizer.labs.dans.knaw.nl/sanitize"
 METADATA_REFINER_URL="https://metadata-refiner.labs.dans.knaw.nl"
-
-BUCKET_NAME = "harvested-metadata"
+SEMANTIC_API_URL="http://semanticapi:80/importdoi"
```

Comment:
Is this url correct?

Reply:
Yes, the service still needs to be deployed. Once it is deployed I will update the settings to match the deployed service.